### ISSUE TITLE
error loading games

### ISSUE THREAD

**Issue by soundsnow** [philj56/gbcc#31]
Nintendo - Game Boy

Bandai TAMA5 not yet supported.
Game de Hakken!! Tamagotchi - Osutchi to Mesutchi (Japan) (SGB Enhanced)

Invalid ram size.
Lasama Chuanqi - Story of Lasama (Taiwan) (Unl)
Super Donkey Kong 3 (Taiwan) (Unl)

Invalid Opcode
GameShark (USA) (Unl)
Joshua & the Battle of Jericho (USA) (Unl)
Mick & Mack as the Global Gladiators (USA) (Proto) (1993-09-27)
Mick & Mack as the Global Gladiators (USA) (Proto) (1994-01-18)
Pro Action Replay (Europe) (Unl)
Pocket Monsters Gin (Japan) (Demo) (Spaceworld 1997) (SGB Enhanced)
Pocket Monsters Gin (Japan) (Demo) (Spaceworld 1997) (SGB Enhanced) (Debug)
Pocket Monsters Kin (Japan) (Demo) (Spaceworld 1997) (SGB Enhanced)
Pocket Monsters Kin (Japan) (Demo) (Spaceworld 1997) (SGB Enhanced) (Debug)
Pokemon - Red Version (Hong Kong) (SGB Enhanced) (Pirate)
seaQuest DSV (USA, Europe) (Beta)

Unrecognised region code.
4 in 1 (Europe) (4B-001, Sachen-Commin) (Unl)
4 in 1 (Europe) (4B-002, Sachen) (Unl)
4 in 1 (Europe) (4B-004, Sachen-Commin) (Unl)
4 in 1 (Europe) (4B-005, Sachen-Commin) (Unl)
4 in 1 (Europe) (4B-006, Sachen) (Unl)
4 in 1 (Europe) (4B-007, Sachen) (Unl)
4 in 1 (Europe) (4B-008, Sachen) (Unl)
4 in 1 (Europe) (4B-009, Sachen) (Unl)
4 in 1 (Taiwan) (En,Zh) (4B-003, Sachen-Commin) (Unl)
8-in-1 (Unknown) (Unl)
Super 69 in 1 (World) (Unl)
Mani 4 in 1 - Bubble Bobble + Elevator Action + Chase H.Q. + Sagaia (China) (En)
Mani 4 in 1 - Genki Bakuhatsu Gambaruger + Zettai Muteki Raijin-Oh + Zoids Densetsu + Miracle Adventure of Esparks - Ushinawareta Seiseki Perivron (China) (Ja)
Mani 4 in 1 - R-Type II + Saigo no Nindou + Ganso!! Yancha Maru + Shisenshou - Match-Mania (China) (Ja)
Mani 4 in 1 - Takahashi Meijin no Bouken-jima II + GB Genjin + Bomber Boy + Milon no Meikyuu Kumikyoku (China) (Ja)
Mani 4 in 1 - Tetris + Alleyway + Yakuman + Tennis (China) (Ja)
Mind Bender (USA) (Beta) (1991-03-04)
Rockman 8 (Taiwan) (Unl)

Nintendo - Game Boy Color

MBC6 not yet supported.
Net de Get - Minigame @ 100 (Japan)

Invalid ram size.
Beast Fighter (Taiwan) (En) (Sachen) (Unl)
Xploder GB (Europe) (v1.2.3E) (GB Compatible) (Unl)
Xploder GB (Germany) (v1.2.2G) (GB Compatible) (Unl)

Invalid Opcode
Bing Yuan Li Xian Ji II (Taiwan) (Unl)
Bomberman 3 (Taiwan) (Unl)
Capcom vs SNK - Millennium Fight 2001 (Taiwan) (En) (Unl)
Chao Ji Ji Qi Ren Da Zhan X - Super Robot War X (Taiwan) (Unl)
Chao Ji Ji Qi Ren Da Zhan X - Super Robot War X (Taiwan) (Unl) (Alt)
Chao Jinhua Shuma Baolong - Zuanshi Ban (Taiwan) (Unl)
Chaoji Gedou 2001 Alpha (Taiwan) (Unl)
Chong Wu Xiao Jing Ling - Jie Jin Ta Zhi Wang (Taiwan) (Unl)
Crash II Advance 2003 (USA) (Unl)
Digimom Sapphii 2003 (Taiwan) (En) (Unl)
Digimon 2 (Taiwan) (En) (Unl)
Digimon 3 Crystal (Taiwan) (En) (Unl)
Digimon 02 4 (Taiwan) (En) (Unl)
Digimon Adventure 6 2002 (Taiwan) (En) (Unl)
Digimon Adventure 2001 (USA) (Unl)
Digimon Crystal II (USA) (Unl)
Dragon Ball Z 3 2002 Fighting (Taiwan) (En) (Unl)
Elang Chuanshuo - Shiji Zhi Zhan (Taiwan) (Unl)
Gangdan Wuyu 2 (Taiwan) (Unl)
Gu Huo Lang II 2003 (Taiwan) (Unl)
Ha Li Bo Te 2 - Xiao Shi De Mi Shi 2003 (Taiwan) (Unl)
Ha Li Xiao Zi Di Er Bu - Mi Shi De Mi (Taiwan) (Unl)
Ha Li Xiao Zi IV 2003 (Taiwan) (Unl)
Kou Dai Guai Shou - Dong Zuo Pian (Taiwan) (En) (Unl)
Kou Dai Guai Shou - Lan Bao Shi 2003 (Taiwan) (Unl)
Langrsr II (Taiwan) (En) (Unl)
Luo Ke Ying Xiong EXE5 (Taiwan) (Unl)
Mythri (USA) (Proto 1) (2000-08-02)
Mythri (USA) (Proto 2) (2001-03-31)
Nuwang Gedou (Taiwan) (Unl)
Pian Wai Zhang Huang Jin Tai Yang - Feng Yin De Yuan Gu Lian Jin Shu (Taiwan) (Unl)
Pocket Monster Carbuncle 2003 (USA) (Unl)
Pokemon - Sapphire Version (USA) (Unl)
Pokemon Diamond (Taiwan) (En) (Unl)
Qi Long Zhu Z 3 (Taiwan) (Unl)
Quan Ba Tian Xia (Taiwan) (Unl)
Rockman DX6 (Taiwan) (Unl)
Rockman DX8 (Taiwan) (En) (Unl)
San Guo Wu Shang 5 (Taiwan) (Unl)
Sanguozhi Wudai (Taiwan) (Unl)
Shu Ma Bao Long - Shui Jing Ban II (Taiwan) (Unl)
Shuihu Shenshou (Taiwan) (Unl) (Alt)
Shuihu Zhuan Zhi Qunmo Fengyun Lu (China) (Unl)
Shuma Baobei DX6 (Taiwan) (Unl)
Shuma Baolong - Diannao Pian (Taiwan) (Zh) (Unl)
Shuma Baolong - Shuijing Ban (Taiwan) (Li Cheng) (Unl)
Shuma Baolong 02 4 (Taiwan) (Zh) (Unl)
Super Marrio Sunshine (Taiwan) (En) (Unl)
Taikong Zhanshi - Jingdian Ban (Taiwan) (Unl)
Taikong Zhanshi DX3 - Zuizhong Huanxiang (Taiwan) (Unl)
The Second Edition Harry Boy - The Secret of the Chamber of Secrets (USA) (Unl)
Yingxiong Jian 2 (Taiwan) (Unl)
Yingxiong Tianxia (Taiwan) (Zh) (Unl)
Yong Zhe Dou E Long VIII (Taiwan) (Unl)
Yue Nan Zhan Yi 3 (Taiwan) (Unl)

Unrecognised region code.
4 in 1 + 8 in 1 (World) (4B-001, 4B-009, 8B-001, Sachen) (Unl)
4 in 1 + 8 in 1 (World) (4B-002, 4B-004, 8B-002, Sachen) (Unl)
4 in 1 + 8 in 1 (World) (4B-005, 4B-006, 8B-003, Sachen) (Unl)
4 in 1 + 8 in 1 (World) (4B-007, 4B-008, 8B-004, Sachen) (Unl)
31 in 1 (Taiwan) (31B-001, Sachen) (Unl)
31-in-1 Mighty Mix (Australia) (31B-001, Sachen) (Unl)
Adventure Digimon 7 2002 (Taiwan) (En) (Unl)
Jurassic Boy 2 (Taiwan) (En) (Rev 1) (Sachen) (Unl)
Jurassic Boy 2 + Thunder Blast Man (Taiwan) (En) (1B-002, 1B-003, Sachen) (Unl)
Rocman X Gold + 4 in 1 (Taiwan) (1B-002, 4B-003, Sachen) (Unl)
Shuihu Shenshou (Taiwan) (Unl)
Star Heritage (Europe) (Proto) (Password Version)
Star Heritage (Europe) (Proto) (SRAM Version)
Street Hero (Taiwan) (En) (1B-004, EB-004, Sachen) (Unl)
Super 6 in 1 (Taiwan) (En,Zh) (6B-001, Sachen) (Unl)
Super 16 in 1 (Taiwan) (En) (Sachen) (Unl)
Thunder Blast Man (Europe) (Sachen) (Unl)

**Comment by philj56** [comment]
Yeah, although it's an error in the ROM, a bad region code shouldn't be a fatal error.
